orangecontrib/spectroscopy/widgets/owspectra.py

```diff
@@ -1128,7 +1259,7 @@ def current_selection():
 
         # redraw whole selection if it increased or decreased over the threshold
         if len(old_sel_ci) <= MAX_THICK_SELECTED < len(new_sel_ci) or \
-           len(old_sel_ci) > MAX_THICK_SELECTED >= len(new_sel_ci):
```

Comment:
Do not modify the code you did not need to edit. And if you do, ALWAYS do it in a separate commit.